test/extended/util/test.go

```diff
@@ -200,6 +200,7 @@ func AnnotateTestSuite() {
 				break
 			}
 		}
+		excludedTestsFilter := regexp.MustCompile(strings.Join(append(excludedTests, fmt.Sprintf(`\[%s:Skipped\]`, e2e.TestContext.Provider)), `|`))
```

Comment:
I think we should only set this if test provider platform is not empty.

Reply:
+1 fixed with https://github.com/openshift/origin/compare/f3da840a3a95e884a362b837c68b6e49321fd6ac...a99a729e4239a3f3ea6c896906cd5f143f0aef96